source/reference/kubectl-minio-plugin/kubectl-minio-tenant.rst

```diff
@@ -0,0 +1,55 @@
+
+.. _kubectl-minio-tenant:
+
+========================
+``kubectl minio tenant``
+========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: kubectl minio tenant
+
+Description
+-----------
+
+.. start-kubectl-minio-tenant-desc
+
+:mc-cmd:`kubectl minio tenant` creates a temporary proxy to forward traffic from the local host machine to the MinIO Operator Console. 
```

Comment:
the description is same as  proxy sub command. is this intentional?
>creates a temporary proxy to forward traffic from the local host machine to the MinIO Operator Console

Reply:
<blush> Thanks for catching that!